themes/frontierline/functions.php

```diff
@@ -728,40 +732,46 @@ function frontierline_save_sidebar_metabox(){
 add_action('admin_init', 'frontierline_register_sidebar_metabox');
 add_action('save_post', 'frontierline_save_sidebar_metabox');
 
+
 /**
- * Display hero images
+ * Hide hero image on single posts
  */
-function frontierline_display_hero($post){
-  $uncropped = get_post_meta($post->ID, '_frontierline_display_hero', true);
+function frontierline_hide_hero($post) {
+    wp_nonce_field(basename( __FILE__ ), '_frontierline_hide_hero_nonce');
+    $hidden = get_post_meta($post->ID, '_frontierline_hide_hero', true);
   ?>
-  <?php if (get_theme_mod('frontierline_no_post_thumbnail') !== '1') : ?>
-    <p><?php _e('If your featured image is important content, you can display it above the post. If this option is unchecked, the image will only be used as a thumbnail and won‘t appear on the post page.', 'frontierline'); ?></p>
-    <label class="selectit" for="frontierline_display_hero">
-      <input type="checkbox" name="_frontierline_display_hero" id="frontierline_display_hero" value="1" <?php if ($uncropped) { ?>checked<?php } ?>>
-      <?php _e('Display featured image', 'frontierline'); ?>
+    <p><?php _e('The image will be used as a thumbnail for this post but won‘t appear on the post page.', 'frontierline'); ?></p>
```

Comment:
This wording needs to be tweaked for the case where summary images are disabled. Maybe add something about social media?